### ISSUE TITLE
[Bug] Null table values break paging

### ISSUE THREAD

**Issue by emigre459** [nielsdejong/neodash#85]
I've noticed if a value anywhere in a Table-type tile is null, it can cause an Error when trying to load the table. This seems page-specific, with the first page of results usually rendering (although I'm not sure of this, admittedly) and later pages with nulls causing the whole tile to error out, requiring it to be re-run (and nulls `coalesce`d away to avoid the errors). Perhaps string replacement triggered by page view of nulls with "None" or "null" would be an ideal fix?

**Comment by nielsdejong** [comment]
Hi @emigre459 ,

I'm trying to reproduce this one but am not getting any errors on my side. Example query with random null & undefined values here:
```
UNWIND range(1,100) as number
RETURN number, 
CASE
  WHEN number % 3 = 0 THEN "test"
  WHEN number % 3 = 1 THEN NULL
  ELSE sqrt(-1)
END AS result
```

I'm on v2.0.11. Does this work for you? If not, could you can engineer an artificial query that reproduces the behaviour?

**Comment by emigre459** [comment]
I'm using v2.0.11 as well. Your query worked fine for me, but when I run this query I still get the error:

```
MATCH (p:Publication)
WHERE p.projectID IS NOT NULL

OPTIONAL MATCH (p)<-[c:CITED_BY]-(p2:Publication)
WITH p.projectID AS ProjectID, 
COUNT(DISTINCT p) AS Papers, 
COUNT(DISTINCT c) AS Citations,
AVG(p.scoreInterDLanguage) AS AvgIDScore

RETURN ProjectID, Papers, Citations, toFloat(Citations)/Papers AS `Citations Per Paper`, AvgIDScore AS `Language Interdisciplinarity`
ORDER BY `Citations Per Paper` DESC
```

The only way to avoid the error is by changing `AvgIDScore AS Language Interdisciplinarity` to `COALESCE(AvgIDScore, 'None') AS Language Interdisciplinarity`. I've attached an image showing the difference between the two queries.

Notably, running the non-COALESCE query in Neo4j Browser doesn't have issues (see other attached image). As you can see from the table entries, I'm simply returning either a float or a null value, so I agree that it's odd your test doesn't reflect what I'm seeing...


![image](https://user-images.githubusercontent.com/11720378/155352611-eef58df5-0a65-4956-b3ae-4b37b4859303.png)

![image](https://user-images.githubusercontent.com/11720378/155352997-59a4be57-2c98-451a-9e05-7fcc73324901.png)

**Comment by nielsdejong** [comment]
https://github.com/nielsdejong/neodash/commit/d23d214b3a64927994be968661477aa780095531
Fixed this here, will include this in the 2.0.12 release due tomorrow.

Thanks!

**Comment by emigre459** [comment]
@nielsdejong I'm afraid we're seeing a different flavor of this bug now in 2.0.12. We've confirmed multiple values from the same calculation that started this Issue that are actually 0.0 when viewed in Neo4j Browser, but now Neodash is rendering them in the table as `null`. It seems to not be erroring out on nulls anymore but 0 is getting null-ified. Do you know why that may be?
